src/util/ped.hpp

```diff
@@ -23,6 +23,10 @@ namespace big::ped
 		return true;
 	}
 
+	bool is_player_in_cutscene(Ped player) { return NETWORK::NETWORK_IS_PLAYER_IN_MP_CUTSCENE(player); }
+
+	bool is_player_in_interior(Ped player) { return (INTERIOR::GET_INTERIOR_FROM_ENTITY(player) > 0) ? true : false; }
```

Comment:
Both of these methods should be `inline` if they're going into a header file

Reply:
originally i had them as inline, removed inline trying to fix the definition error and never did it